lib/roo_on_rails/sidekiq/settings.rb

```diff
@@ -1,16 +1,19 @@
 module RooOnRails
   module Sidekiq
     class Settings
+      DEFAULT_QUEUES = %w(
+        monitoring
+        realtime
+        within1minute
+        within5minutes
+        within30minutes
+        within1hour
+        within1day
+      ).freeze
+
       def self.queues
-        %w(
-          monitoring
-          realtime
-          within1minute
-          within5minutes
-          within30minutes
-          within1hour
-          within1day
-        ).freeze
+        env_key = 'SIDEKIQ_QUEUES'
+        ENV.key?(env_key) ? ENV[env_key].split(',') : DEFAULT_QUEUES
```

Comment:
Also, you'll need to sort the queues by latency otherwise they'll be read in the wrong order. You _could_ rely on people always putting a 1hr queue before a 1day queue, but I probably wouldn't. If you _do_ rely on that, then you should at least document that lower latency queues must appear first.